byoqm/runner/test_runner.py

```diff
@@ -15,6 +16,7 @@ def setUp(self):
         _OUTPUT_FOLDER.mkdir(parents=True, exist_ok=True)
 
         self._runner = Runner("no_cpd", Path("byoqm/"), _OUTPUT_FOLDER, True, "python")
+        logging.disable()
```

Comment:
I think we should leave logging on. Is it not fine to be able to see potential errors when testing?

Reply:
<img width="918" alt="image" src="https://user-images.githubusercontent.com/60236750/225848428-3be76470-2d80-439b-8f90-fea65d6d00ee.png">

u sure?

Reply:
I think its valuable to be able to see logging to debug failing tests, but I can also see the argument that you only need to remove one line to get it back.